### DISCUSSION TITLE
Init error of v0.7.1.6 version using centralMaven artifact repository

### DISCUSSION THREAD

**Discussion by HACore** [euphony-io/euphony#34] [General]
**Environment**
> **Device** : Pixel 3a (Android Emulator)
> **OS** : Android 10
> **Android Studio** : Arctic Fox | 2020.3.1

**Describe the bug**
When inserting `EuRxManager` object into the code, there's a problem that the app is terminated with fatal exception.
Initializing this object causes `NoClassDefFoundError`.

![image](https://user-images.githubusercontent.com/58473522/130658999-ae36eb22-0d90-48cc-97b4-e2cedd268835.png)

The imported `Euphony` is 0.7.1.6 version on the centralMaven artifact repository
```
implementation 'co.jbear.lib:euphony:0.7.1.6'
```

While looking for a way to resolve this issue, I found out that this issue may have been caused by setting 'testCoverageEnabled true' in library module.
(ref. [Setting 'testCoverageEnabled true' in library module crashes app on start-up](https://issuetracker.google.com/issues/37091130))

Fortunately, it was reflected in a084f06 recently as I looked at commit history and from the next version, it seems that this issue will not be there.
But for the unlucky person who import `co.jbear.lib:euphony:0.7.1.6` like me, I thought I should report the solution to this issue.
I hope this report helps someone suffering🤣

And also, thanks to [@designe comment](17#issuecomment-902351596), I got to know version prior to 0.8 is working well on jcenter.
> > I have no idea what's different
> > ```
> > implementation 'euphony.lib:euphony:0.7.1.6'
> > ```
> > 
> > and
> > ```
> > implementation 'co.jbear.lib:euphony:0.7.1.6'
> > ```
> >   
> > Should we add the
> > ```
> > implementation 'co.jbear.lib:euphony:0.7.1.6'
> > ??
> > ```
> 
> library content is not different.
> but,
> euphony.lib:euphony:0.7.1.6 is on the jcenter
> and co.jbear.lib:euphony:0.7.1.6 is on the centralMaven
> 
> We're going to use centralMaven after 0.8 version, because jcenter artifact repository was sunset last year :(
> Previous uploaded version is working well on jcenter.
> So, don't care about centralMaven version in v0.7.1.6

So the simplest solution is to use `euphony.lib:euphony:0.7.1.6`!
```
implementation 'euphony.lib:euphony:0.7.1.6'
```

**Comment by designe** [comment]
@HACore Thanks for your report!!
It will be great information for beginners!
When v0.8 version will be launched, Let me double check this issue #30

**Comment by HACore** [comment]
I'm looking forward to v0.8 version!
Cheers to the work!! 💪

**Comment by designe** [comment]
@HACore Let me convert this issue to discussions!
